x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_chart_fixed_axis.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  Axis,
+  BarSeries,
+  BarStyleAccessor,
+  Chart,
+  DomainRange,
+  Position,
+  ScaleType,
+  Settings,
+  TickFormatter,
+  TooltipType,
+} from '@elastic/charts';
+import { useChartTheme } from '../../../../../hooks/use_chart_theme';
+import { WaterfallChartFixedAxisContainer } from './styles';
+
+interface Props {
+  tickFormat: TickFormatter;
+  domain: DomainRange;
+  barStyleAccessor: BarStyleAccessor;
+}
+
+export const WaterfallChartFixedAxis = ({ tickFormat, domain, barStyleAccessor }: Props) => {
+  const theme = useChartTheme();
+
+  return (
+    <WaterfallChartFixedAxisContainer>
```

Comment:
Should we add an aria-hidden here?

Reply:
@katrin-freihofner The entire chart, including this axis, is currently inaccessible. Being a canvas image, the screen reader literally just reads out image. We should consider having a way to represent the chart as visually hidden accessible text. It would also be good to check with the Charts team about how they are thinking about accessibility.